libraries/AP_SerialManager/AP_SerialManager.h

```diff
@@ -87,7 +91,8 @@ class AP_SerialManager {
         SerialProtocol_FrSky_SPort_Passthrough = 10, // FrSky SPort Passthrough (OpenTX) protocol (X-receivers)
         SerialProtocol_Lidar360 = 11,                // Lightware SF40C, TeraRanger Tower or RPLidarA2
         SerialProtocol_Aerotenna_uLanding      = 12, // Ulanding support
-        SerialProtocol_Beacon = 13
+        SerialProtocol_Beacon = 13,
+		SerialProtocol_Volz = 14
```

Comment:
You need to add this to all the SERIALx_PROTOCOL parameter descriptions (the @values tag).

Nitpick since I'm writing a comment on this line the indentation level for SerialProtocol_Volz is different.

Reply:
added comment, indentation fixed